src/Workspaces/CSharp/Portable/Formatting/Rules/SuppressFormattingRule.cs

```diff
@@ -286,6 +289,109 @@ private void AddStatementExceptBlockSuppressOperations(List<SuppressOperation> l
             AddSuppressWrappingIfOnSingleLineOperation(list, firstToken, lastToken);
         }
 
+        private void AddFormatSuppressOperations(List<SuppressOperation> list, SyntaxNode node)
+        {
+            foreach (var child in node.ChildNodesAndTokens())
+            {
+                if (!child.IsToken)
+                {
+                    continue;
+                }
+
+                ProcessTriviaList(list, child.AsToken().LeadingTrivia);
+                ProcessTriviaList(list, child.AsToken().TrailingTrivia);
+            }
+
+            // Local functions
```

Comment:
nit: consider explicit `return` above this to make it clear that there's no more 'main body code' that comes after the local functions.

Reply:
Done!